code/modules/admin/permissionedit.dm

```diff
@@ -195,19 +198,27 @@
 		GLOB.admin_ranks += R
 	if(use_db)
 		new_rank = sanitizeSQL(new_rank)
-		if(!R)
-			var/datum/DBQuery/query_add_rank = SSdbcore.NewQuery("INSERT INTO [format_table_name("admin_ranks")] (rank, flags, exclude_flags, can_edit_rights) VALUES ('[new_rank]', '0', '0', '0')")
+		//if a player was tempminned before having a permanent change made to their rank they won't yet be in the db
+		var/old_rank
+		var/datum/DBQuery/query_admin_in_db = SSdbcore.NewQuery("SELECT rank FROM [format_table_name("admin")] WHERE ckey = '[admin_ckey]'")
```

Comment:
sanitize you madlad!

Reply:
This value is already sanitized before being passed down.

Reply:
pretty sure it'd be trivial for kevinz to find a way to procall this though

Reply:
wait, the safteies, nvm